python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -57,19 +58,39 @@ class AssembleCoaddConfig(CoaddBaseTask.ConfigClass):
         length=2,
         default=(2000, 2000),
     )
+    statistic = pexConfig.ChoiceField(
+        dtype=str,
+        doc="Main stacking statistic for aggregating over the epochs.",
+        default="MEANCLIP",
+        allowed={
+            "MEAN": "unclipped mean",
+            "MEANCLIP": "N-sigma clipped mean (N set by sigmaClip)",
+            "MEDIAN": "median",
+            "NPOINT": "number of epochs",
+            "STDEV": "standard deviation",
+            "VARIANCE": "variance",
+            "IQRANGE": "inter-quartile range",
+            "STDEVCLIP": "N-sigma clipped stdev (N set by sigmaClip)",
+            "VARIANCECLIP": "N-sigma clipped variance (N set by sigmaClip)",
+            "MIN": "minimum",
+            "MAX": "maximum",
+            "SUM": "sum",
+            "MEANSQUARE": "mean of square of  pixel values",
+        }
+    )
```

Comment:
Yeah, writing out the allowed options here really sucks, doesn't it.  I liked the idea of the options being exposed in the resulting config files.  What I really want is the stats class to give me a dictionary of available statistics for stacking. I talked this over with @pschella, who says the new API will offer that.  I ended up specifying them explicitly because:
1) Not all statistics area available for stacking: NOTHING, ERROR, ORMASK make no sense in a coadd and stackStatistics raises InvalidParameter errors or segfault (in the case of ORMASK)
2) The only documentation for the options in the `Property enum`  is in code comments. There's no way to retrieve them.

Without exposing the options, I could add to the config's `validate`: 
```
        UNSTACKABLE_STATS = ['NOTHING', 'ERROR', 'ORMASK']
        if not hasattr(afwMath.Property, self.statistic) or self.statistic in UNSTACKABLE_STATS:
            stackableStats = [str(k) for k in afwMath.Property.__members__.keys() 
                         if str(k) not in UNSTACKABLE_STATS]
            raise ValueError("statistic %s is not allowed. Please choose one of %s."
                             % (self.statistic, stackableStats))
```
sufficient solution?